app/services/reinstate_deferred_offer.rb

```diff
@@ -0,0 +1,26 @@
+class ReinstateDeferredOffer
```

Comment:
can we use the wording confirm deferred offer instead, as we are moving away from reinstate?